app/src/components/inputs/inputDropdown/inputDropdown.jsx

```diff
@@ -34,7 +34,6 @@ export class InputDropdown extends Component {
     multiple: PropTypes.bool,
     selectAll: PropTypes.bool,
     disabled: PropTypes.bool,
-    active: PropTypes.bool,
```

Comment:
so dropdown loses the way to be controlled.
Is it expected?

Reply:
Yes, because now dropdown will be controlled by state